cmd/rosetta-dispatcher-server/README.md

```diff
@@ -0,0 +1,10 @@
+# Rosetta Dispatcher Server
+
+## Description
+
+The Rosetta Dispatcher Server is a simple proxy/dispatcher which forwards the HTTP request
+to appropriate spork based on block height in a  provided spork list
+
+Current implementation of Rosetta API require block height (called Index in Rosetta nomenclature) to be provided
+and this allows proxy to simply check height boundaries. If this changes, as Rosetta API permits using only block hash
+a new implementation of routing mechanism should be created
```

Comment:
```suggestion
The current implementation of Rosetta API requires requests to include block height (called Index in Rosetta nomenclature.)
This allows the proxy to check height boundaries to redirect the request to the matching spork.
If we switch to make requests be based on block hashes instead of block heights, we will need to implement a new routine mechanism.
```